.prettierrc.js

```diff
@@ -0,0 +1,3 @@
+module.exports = {
+  ...require('gts/.prettierrc.json'),
```

Comment:
Just by interest, I used to not use the semicolons at the end of the line. Was that discussed somewhere if it's desired to use or not?
[directed-stake .prettierrs.js config ](https://github.com/marinade-finance/directed-stake/blob/main/.prettierrc.js)

Reply:
Not really, I just used gts standard setup as I saw you also used it in the directed stake sdk and it was really easy to set up. If wished we can certainly change this.